graylog2-server/src/main/java/org/graylog2/indexer/searches/Searches.java

```diff
@@ -204,13 +204,15 @@ public CountResult count(String query, TimeRange range, String filter) {
         } else {
             searchSourceBuilder = filteredSearchRequest(query, filter, range);
         }
-        searchSourceBuilder.size(0);
 
-        final Set<String> indices = determineAffectedIndices(range, filter);
+        final Set<String> affectedIndices = determineAffectedIndices(range, filter);
+        if (affectedIndices.size() == 0) {
```

Comment:
Aesthetic nitpick: This could be `affectedIndices.isEmpty()` to make the intention 100% clear beyond any doubt.